packages/realm-web/src/User.ts

```diff
@@ -153,13 +114,7 @@ export class User implements Realm.User {
     private _state: Realm.UserState;
     private transport: AuthenticatedTransport;
```

Comment:
The private fields with an `_` prefix is only because the class also has a public getter overloading the name. It could be called `internalState` or something else. But I actually think this specific fields goes away in one of the later PRs.

With regards to JS private fields. It's valuable to keep an eye on, but I don't think [the adoption in browsers](https://caniuse.com/#feat=mdn-javascript_classes_private_class_fields) are good enough for us to rely on. And I don't know the state of polyfils for this?

Reply:
I think TS handles that, but I'm not sure (as in I haven't tested it) https://github.com/microsoft/TypeScript/pull/30829

Reply:
Oh ...